_content/tour/por/algorithms-strings.article

```diff
@@ -1,67 +1,66 @@
-String Operations
-This section provides examples that perform string operations.
+Operações com Strings
+Esta seção fornece exemplos que realizam operações em strings.
 
-* Permutation
+* Permutação
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Assista ao Vídeo]]
+- Precisa de Assistência Financeira? Use Nosso [[https://www.ardanlabs.com/scholarship/][Formulário de Bolsa de estudos]]
 
-The sample program implements a check to see if a string is a
-permutation or not.
+O programa exemplo implementa uma verificação para ver se uma string é ou não uma permutação.
 
-- See more at [[https://en.wikipedia.org/wiki/Permutation][https://en.wikipedia.org/wiki/Permutation]]
 
-*Diagram*
+- Veja mais em [[https://pt.wikipedia.org/wiki/Permutação][https://pt.wikipedia.org/wiki/Permutação]]
 
-    The permutation of a string is the set of all strings that contains the
-    same characters without the order of the arrangement of the characters
-    mattering.
+*Diagrama*
+
+    A permutação de uma string é o conjunto de todas as strings em que
+    contêm os mesmos caracteres sem importar a ordem da combinação de caracteres
 
     ┌───┐┌───┐┌───┐
     │ G ││ O ││ D │ ──────┐
-    └───┘└───┘└───┘       │ ────▷ Permutation
+    └───┘└───┘└───┘       │ ────▷ Permutação
     ┌───┐┌───┐┌───┐       │
     │ D ││ O ││ G │ ──────┘
     └───┘└───┘└───┘
 
     ┌───┐┌───┐┌───┐
     │ G ││ O ││ D │ ──────┐
-    └───┘└───┘└───┘       │ ────▷ NO
+    └───┘└───┘└───┘       │ ────▷ NÃO
     ┌───┐┌───┐            │
     │ D ││ O │      ──────┘
     └───┘└───┘
 
     ┌───┐┌───┐┌───┐
     │ 1 ││ 0 ││ 0 │ ──────┐
-    └───┘└───┘└───┘       │ ────▷ Permutation
+    └───┘└───┘└───┘       │ ────▷ Permutação
     ┌───┐┌───┐┌───┐       │
     │ 0 ││ 0 ││ 1 │ ──────┘
     └───┘└───┘└───┘
 
 .play algorithms/strings/permutation.go
 
-* Palindrome
+* Palíndromo
 
-The sample program implements a check to see if a string is a
-palindrome or not.
+O programa exemplo implementa uma verificação para ver se a string
+é ou não palíndromo. 
 
-- See more at [[https://en.wikipedia.org/wiki/Palindrome][https://en.wikipedia.org/wiki/Palindrome]]
+- Veja mais em [[https://pt.wikipedia.org/wiki/Palíndromo][https://pt.wikipedia.org/wiki/Palíndromo]]
```

Comment:
I don't think we should change the link.

Reply:
I have the same thought @BFostek let's keep the links.